server/migrations/1637909008629-CreateAuditLogs.ts

```diff
@@ -43,6 +43,11 @@ export class CreateAuditLogs1637909008629 implements MigrationInterface {
             type: 'varchar',
             isNullable: true,
           },
+          {
```

Comment:
Should not add migrations like this, should create new migration file and add column